pyproject.toml

```diff
@@ -46,7 +46,6 @@ dependencies = [
     "websocket >= 0.2.1",
     "websockets >= 12.0",
     "zarr >=2.15.0,<3.0.0",
-    "pyqt5 >=5.15.4",
```

Comment:
I think you should add `qtpy` because you import from it.

Reply:
It's a transitive dependency right now, so everything works.

Reply:
thanks, @psobolewskiPhD, you did mention the issue, and I forgot to include it
I'm fixing the errors in a different PR and this should be good soon.

Happy holidays!